sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1Test.java

```diff
@@ -645,6 +646,39 @@ private void datastoreWriterFnTest(int numMutations) throws Exception {
   }
 
   /**
+   * Tests {@link DatastoreWriterFn} with large entities that need to be split into more batches.
+   */
+  @Test
+  public void testDatatoreWriterFnWithLargeEntities() throws Exception {
+    List<Mutation> mutations = new ArrayList<>();
+    for (int i = 0; i < 12; ++i) {
```

Comment:
Can this be computed from `DATASTORE_BATCH_UPDATE_BYTES_LIMIT`, so that this test is robust against any changes in the value?